src/collectors/netstat/netstat.py

```diff
@@ -46,26 +46,19 @@ def collect(self):
         Overrides the Collector.collect method
         """
 
-        content = self._load()
-        result = dict((self.STATE[num], 0) for num in self.STATE)
+        result = {self.STATE[num]: 0 for num in self.STATE}
```

Comment:
@shortdudey123 Hi!

I got feedback that in my patch, replacing dict() to {} led to incompatibility with python 2.6, which is a system one for CentOS6 and the collector breaks there.

If it seems important to you to maintain backward compatibility with old version (I could not find which versions are supported anywhere), then I will roll back this line. What do you think of it?

Thanks in advance for your reply.